Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -339,13 +392,15 @@ theorem tendsto_atBot_intervalIntegral_of_pos (h₀ : 0 < ∫ x in (0)..T, g x)
 tends to `∞` as `t` tends to `∞`. -/
 theorem tendsto_atTop_intervalIntegral_of_pos' (h₀ : ∀ x, 0 < g x) (hT : 0 < T) :
     Tendsto (fun t => ∫ x in (0)..t, g x) atTop atTop :=
-  hg.tendsto_atTop_intervalIntegral_of_pos h_int (intervalIntegral_pos_of_pos (h_int 0 T) h₀ hT) hT
+  hg.tendsto_atTop_intervalIntegral_of_pos h_int (intervalIntegral_pos_of_pos
+    (hg.intervalIntegrable₀ hT h_int 0 T) h₀ hT) hT
```

Comment:
```suggestion
  hg.tendsto_atTop_intervalIntegral_of_pos (intervalIntegral_pos_of_pos h_int h₀ hT) hT
```
No need to apply your theorem to deduce that the function is integrable on `[0,T]` from the fact that it's integrable on `[0, T]` :-)

Reply:
Rats. But at least it's not wrong, either.